apps/server/src/services/runtime-service/rundownService.utils.ts

```diff
@@ -26,8 +26,7 @@ export function getShouldTimerUpdate(previousValue: number, currentValue: MaybeN
     return false;
   }
   // we avoid trigger ahead since it can cause duplicate triggers
-  // we force the timer value to be negative because we need a ceiling reduction
-  const shouldUpdateTimer = millisToSeconds(-currentValue) !== millisToSeconds(-previousValue);
```

Comment:
with the way we push updates now we end up with a timer that is almost constantly 980ms ahead of time
as @kev-ac  notes this is the correct rounding
but since we trigger updates using `millisToSeconds`  that means that we now send updates exactly when the error is the biggest